doc/source/dev/dev_quickstart.rst

```diff
@@ -16,8 +16,9 @@ steps to start contributing:
 2. **Build SciPy**
 
    SciPy uses compiled code for speed, which means you might need extra
-   dependencies to complete this step depending on your system - see
-   :ref:`building-from-source`.
+   dependencies to complete this step depending on your system. For detailed
+   instructions on building SciPy from source, please refer to the 
+   `Building From Source for SciPy Development <https://scipy.github.io/devdocs/building/index.html#building-from-source-for-scipy-development>`_ guide.
```

Comment:
Wouldn't this only reduce maintenance burden if we actually delete the older/duplicated instructions about `Conda env` and `Virtual env` below? If I understand Lucas correctly, the point is that we longer want to maintain the extra/older instructions below.

Reply:
Alright, got it. So, you want me to remove the duplicated Conda and Virtual environment instructions entirely and just direct people to the Building SciPy From Source guide? Let me know if that's the plan, and I'll make those changes.

Reply:
Yeah, what I had in mind was replacing the section that starts with "execute the
following commands" with a link to the building from source guide. I suppose it is already linked on line 20 though, so maybe there is a better way to restructure the page! The main thing is to remove the conda/venv instructions from the page.

Reply:
Ok, there we go.